wscript

```diff
@@ -180,7 +180,6 @@ def build_lib(bld):
             src/lib/net/message.cc
 
             src/lib/rules/actions.cc
-            src/lib/rules/messenger.cc
```

Comment:
Let's remove that file altogether: https://github.com/prologin/stechec2/blob/master/src/lib/rules/messenger.cc

Reply:
Disregard that comment -- this PR already deletes it.